docs/source/posts/2021/2021-06-28-week-4-sajag.rst

```diff
@@ -0,0 +1,48 @@
+Third week of coding!
+=====================
+
+.. post:: June 28 2021
+   :author: Sajag Swami
+   :tags: google
+   :category: gsoc
+
+Welcome to the fourth weekly check-in. I'll be sharing my progress for the third week of coding.
+
+What did you do this week?
+--------------------------
+
+I made a document with code snippets and visuals to show how one can use
+some vtk classes in python for molecular visualization. Classes of
+interest -
+
+-  vtkMolecule (store atomic information about the molecule)
+-  vtkSimpleBondPerceiver (calculate bonding info for a vtkMolecule)
+-  vtkMoleculeMapper (mapper to draw vtkMolecule object)
+-  vtkPeriodicTable (stores chemical data sourced from the Blue Obelisk
+   Data).
+
+Link to the document - `Molecular_viz_vtk`_. In addition to the
```

Comment:
you might need to convert the google drive to something else

Reply:
Can you please provide any suggestions regarding this?

Reply:
```suggestion
Link to the document: `Molecular_viz_vtk`_. In addition to the
```